README.md

```diff
@@ -49,7 +49,21 @@ If you'd like to add a new sample to this repository, please follow the below st
     - `thumbnailPath`: The path to the thumbnail image of the sample. This should be the path to the thumbnail image relative to the `repositoryUrl` directory. It is recommended to add the thumbnail to the `.samples/icons` directory and refer it here.
     - `documentationPath`: The path to the documentation/README.md of the sample. This should be the path to the documentation relative to the `repositoryUrl` directory.
     - `tags`: Tags for the sample. This should be a list of strings.
-4. Submit a pull request to this repository.
+
+4. If you're adding a Quick Deployable Sample, follow these additional steps:
```

Comment:
Shall we also include "if not jump to 5" in somewhere here.

Reply:
We have the word "additional". Also, Step 5 is obvious as it just say to put the pull request.